Resources/public/js/views/ezsearch-searchlistview.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ezsearch-searchlistview', function (Y) {
+    "use strict";
+  
+    Y.namespace('eZSearch');
+
+    Y.eZSearch.SearchListView = Y.Base.create('ezsearchSearchListView', Y.eZ.TemplateBasedView, [], {
+        
+        initializer: function () {
+        },
+        
+        render: function () {
+            this._setResultItems();
+            this.get('container').setHTML(this.template({
+                searchResultItems: this.get('searchResultItems'),
+                columns: this._getColumns(),
+            }));
+            return this;
+        },
+
+        /**
+         * Returns an array of objects describing the columns to add to the
+         * list. Each object contains an `identifier` and a `name`.
+         *
+         * @method _getColumns
+         * @protected
+         * @return Array
+         */
+        _getColumns: function () {
+            return this.get('displayedProperties').map(function (identifier) {
+                return {
+                    name: this.get('propertyNames')[identifier],
+                    identifier: identifier,
+                };
+            }, this);
+        },
+        
+        _setResultItems: function () {
+            var searchResultItems = [];
+
+            if ( !this.get('searchResultList') ) {
+                return this.get('searchResultList');
+            }
+
+            this.get('searchResultList').forEach(function (struct) {
+                searchResultItems.push({
+                    name: struct.content.get('name'),
+                    lastModificationDate: struct.content.get('lastModificationDate').toLocaleDateString(
+                        'en', {year: "numeric", month: "short", day: "numeric"}
+                    ) + ' ' + struct.content.get('lastModificationDate').toLocaleTimeString(
+                        'en', {hour: '2-digit', minute: '2-digit', hour12: true}
+                    ),
+                    contentType: struct.contentType.get('names')['eng-GB'],
+                    translations: struct.content.get('currentVersion').getTranslationsList().join(', '),
+                });
+            }, this);
+            this._set('searchResultItems', searchResultItems);
+        },
+
+    }, {
+        ATTRS: {
+            searchResultList: {},
+            searchResultItems: {},
+            propertyNames: {
+                value: {
+                    'name': 'Name',
+                    'lastModificationDate': 'Modified',
+                    'contentType': 'Content type',
+                    'translations': 'Translations',
+                }
+            },
+            displayedProperties: {
+                value: ['name', 'lastModificationDate', 'contentType', 'translations'],
```

Comment:
this gives the impression that you just have to add something here to get a new column in the page but that's not true, better (for now) hard code the columns in the template